ci/run_app_creator.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+bundle install
+bundle exec cap android:create p42app p42app
```

Comment:
should this `p42app` be an environment variable also? it's used quite a lot here and in `codemagic.yaml`

Reply:
Well, from coding best practices maybe yes, but IMHO it is very unlikely that we need to change this value everywhere very often (or at all). And I guess it can make testing this script locally harder.
One option is to use a an environment variable, but also have a default value. Works here in the script so that it allows to both locally test easily, and specify values in the CI. But making this both in YAML and here takes some work and not sure if this is worth it.
I personally would leave it as it is atm.

Reply:
I think it is fine to leave it as is. Most likely, it will never be changed